### PR TITLE
docs: Add write support documentations

### PR DESCRIPTION
**Pull request by jiangmichaellll**
This needs to wait until https://github.com/googleapis/java-pubsublite-spark/pull/131 and https://github.com/googleapis/java-pubsublite-spark/pull/122 are checked in.

### CODE REVIEW

**Comment by jiangmichaellll** [comment]
Now that both PRs are checked in it's ready for review.

File: .readme-partials.yaml [hunk 1]
```diff
@@ -28,20 +28,36 @@ custom_content: |
...
     df = spark.readStream \
-      .option("pubsublite.subscription", "projects/$PROJECT_NUMBER/locations/$LOCATION/subscriptions/$SUBSCRIPTION_ID")
       .format("pubsublite") \
+      .option("pubsublite.subscription", "projects/$PROJECT_NUMBER/locations/$LOCATION/subscriptions/$SUBSCRIPTION_ID") \
       .load
     ```
 
     Note that the connector supports both MicroBatch Processing and [Continuous Processing](https://spark.apache.org/docs/latest/structured-streaming-programming-guide.html#continuous-processing).
 
+  ### Writing data to Pub/Sub Lite
```
Comment (anguillanneuf) [inline .readme-partials.yaml:46]:
```suggestion
  ### Writing data to Pub/Sub Lite (Python)
```

File: .readme-partials.yaml [hunk 2]
```diff
@@ -28,20 +28,36 @@ custom_content: |
 
   ## Usage
 
+  ### Samples
+
+    There are 3 samples (word count, simple write, simple read) under [samples](https://github.com/googleapis/java-pubsublite-spark/tree/master/samples) that shows using the connector inside Dataproc.
```
Comment (anguillanneuf) [inline .readme-partials.yaml:33]:
```suggestion
    There are 3 Python samples (word count, simple write, simple read) under [samples](https://github.com/googleapis/java-pubsublite-spark/tree/master/samples) that shows using the connector inside Dataproc.
```

File: .readme-partials.yaml (hunk 1 above)
Comment (jiangmichaellll) [reply .readme-partials.yaml:54]: Added java as well.

File: .readme-partials.yaml (hunk 2 above)
Comment (jiangmichaellll) [reply .readme-partials.yaml]: made it clear that it's java samples.

**Review by anguillanneuf — CHANGES_REQUESTED** [review]
Could you spell out "Here's an example in Python" and "Here's an example in Java"? It's because it won't be clear in the Preview form. LGTM after that.

**Review by anguillanneuf — APPROVED** [review]

**Pull request merged**